`SelectionChangedEventArgs.CurrentSelection` is obsolete

SharedMauiXamlStyles\src\SharedMauiXamlStylesLibrary.Syncfusion\Controls\MultiSelectComboBox.cs(17,17,17,35): warning CS0618: "SelectionChangedEventArgs.CurrentSelection" ist veraltet: "This property is obsolete. Use AddedItems property instead."

--- a/src/SharedMauiXamlStylesLibrary.Syncfusion/Controls/MultiSelectAutoComplete.cs
+++ b/src/SharedMauiXamlStylesLibrary.Syncfusion/Controls/MultiSelectAutoComplete.cs
@@ -14,14 +14,18 @@ namespace AndreasReitberger.Shared.Syncfusion.Controls
 
         void MultiSelectAutoComplete_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            if (e.CurrentSelection is not IList list)
-            {
-                // Single item (if token is set to 'none')
-                list = new ArrayList
-                {
-                    e.CurrentSelection
-                };
-            }
+            ArrayList list = new(SelectedItemsList);
+            // Add new items
+            if (e?.AddedItems is not null)
+                foreach (var item in e?.AddedItems)
+                    if (!list.Contains(item))
+                        list.Add(item);
+            // Remove items
+            if (e?.RemovedItems is not null)
+                foreach (var item in e?.RemovedItems)
+                    if (list.Contains(item))
+                        list.Remove(item);
+            // Update list
             SelectedItemsList = list;
         }
 

--- a/src/SharedMauiXamlStylesLibrary.Syncfusion/Controls/MultiSelectComboBox.cs
+++ b/src/SharedMauiXamlStylesLibrary.Syncfusion/Controls/MultiSelectComboBox.cs
@@ -14,14 +14,18 @@ namespace AndreasReitberger.Shared.Syncfusion.Controls
 
         void ComboBoxItemSelect_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            if (e.CurrentSelection is not IList list)
-            {
-                // Single item (if token is set to 'none')
-                list = new ArrayList
-                {
-                    e.CurrentSelection
-                };
-            }
+            ArrayList list = new(SelectedItemsList);
+            // Add new items
+            if (e?.AddedItems is not null)
+                foreach (var item in e?.AddedItems)
+                    if (!list.Contains(item))
+                        list.Add(item);
+            // Remove items
+            if (e?.RemovedItems is not null)
+                foreach (var item in e?.RemovedItems)
+                    if (list.Contains(item))
+                        list.Remove(item);
+            // Update list
             SelectedItemsList = list;
         }
 

